packages/discovery-provider/src/tasks/entity_manager/entities/track.py

```diff
@@ -292,11 +292,21 @@ def populate_track_record_metadata(track_record: Track, track_metadata, handle,
                 # casting to string because datetime doesn't work for some reason
                 parsed_release_date = parse_release_date(track_metadata["release_date"])
                 # postgres will convert to a timestamp
+
+                if parsed_release_date > datetime.now().astimezone(timezone.utc):
```

Comment:
does somewhere also ignore if the parsed_release_date isn't in valid format? more of a malicious case so just curious

Reply:
ooh true yea tests show i forgot to add a null check here, but yea parsed_release_date will be null if it wasn't in the right format